modules/react/popup/stories/Popup.stories.mdx

```diff
@@ -0,0 +1,318 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Popup} from '@workday/canvas-kit-react/popup';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {Simple} from './examples/Simple';
+import {InitialFocus} from './examples/InitialFocus';
+import {OpenWithoutModel} from './examples/OpenWithoutModel';
+import {ManualControl} from './examples/ManualControl';
+import {MultiplePopups} from './examples/MultiplePopups';
+import {NestedPopups} from './examples/NestedPopups';
+import {RTL} from './examples/RTL';
+import {UsePopup} from './examples/UsePopup';
+import {PopupModelConfigComponent, PopupStateComponent, PopupEventsComponent} from './PopupModel';
+
+<Meta title="Components/Popups/Popup/React" component={Popup} />
+
+# Canvas Kit Popups
+
+A "popup" is a classification for a type of stacked UI element that appears "on top" of statically
+positioned content. Tooltips, Modals, Dropdown menus, etc are all examples of "popups". Canvas Kit
+has a "stack manager" system for managing these stacked UIs. Different types of popups have
+different requirements of behavior for UX and accessibility - we can call them behaviors,
+capabilities, or traits. Canvas Kit comes with a number of [behaviors](#hooks) in the form of React
+Hooks.
+
+You should use the most semantic component for your use-case before using `Popup` directly, like
+`Modal`, which already has the correct behaviors built-in. If no component already exists, you can
+use `Popup` and use our [hooks](#hooks). The `Popup` component comes with a `Popup.Popper` component
+that positions a popup using [PopperJS](https://popper.js.org/) that registers a popup with the
+`PopupStack` automatically and sets the popup model's `placement` property. `Popup.Popper` component
+and hooks work with the stack management system for correct rendering and accessibility behavior. If
+you cannot use `Popup.Popper`, use the [usePopupStack](#usepoupstack) hook to properly register and
+deregister the popup at the correct time. If you cannot use our hooks, consider upgrading your
+component to use Hooks. If you cannot do that, you'll have to look up the `PopupStack` package for
```

Comment:
> `PopupStack` package

Link?

Reply:
I think we should link between documents, but we don't have a way to do that well yet. Do we link to other stories? If yes, how do we know when those links break?